src/masternodeman.cpp

```diff
@@ -862,8 +888,18 @@ void CMasternodeMan::ProcessMessage(CNode* pfrom, std::string& strCommand, CData
         if (!masternodeSync.IsSynced()) return;
 
         CTxIn vin;
+        CBloomFilter filter;
         vRecv >> vin;
 
+        if(pfrom->nVersion >= DSEG_FILTER_VERSION) {
+            vRecv >> filter;
+            filter.UpdateEmptyFull();
```

Comment:
I'm not convinced :) https://github.com/dashpay/dash/blob/master/src/bloom.h#L71